packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6562,6 +6560,64 @@ class RepaintBoundary extends SingleChildRenderObjectWidget {
 /// the semantics layer (and thus e.g. accessibility tools). If
 /// [ignoringSemantics] is null, it uses the value of [ignoring].
 ///
+/// {@youtube 560 315 https://www.youtube.com/watch?v=qV9pqHWxYgI}
+///
+/// {@tool dartpad --template=stateful_widget_material}
+/// The following sample has an [IgnorePointer] widget wrapping the `Column`
+/// which contains a button. 
+/// When setting [ignoring] to `true` anything inside the `Column` will 
+/// not be pressable or tapable. When setting [ignoring] to `false` anything
+/// inside the `Column` will be pressable or tapable.
+///
+/// ```dart
+/// bool ignoring = false;
+/// void setIgnoring(bool newValue) {
+///   setState(() {
+///     ignoring = newValue;
+///   });
+/// }
+/// 
+/// @override
+/// Widget build(BuildContext context) {
+///   return Scaffold(
+///     appBar: AppBar(
+///       centerTitle: true,
+///       title: ElevatedButton(
+///         onPressed: () {
+///           setIgnoring(!ignoring);
+///         },
+///         child: Text(
+///           ignoring ? 'Set Ignoring To False' : 'Set Ignoring To True',
+///         ),
+///       ),
+///     ),
+///     body: Center(
+///       child: IgnorePointer(
+///         ignoring: ignoring,
+///         child: Column(
+///           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+///           children: <Widget>[
+///             Text(
+///               'Ignoring status: ${ignoring ? 'TRUE' : 'FALSE'}',
+///             ),
+///             ElevatedButton(
+///               onPressed: () {},
+///               child: Text(
+///                 'CLICK ME!',
+///                 style: TextStyle(
+///                   fontSize: 50,
+///                 ),
```

Comment:
It's a best-practice to use the Material theme for text styles. It's also a little too big :)

```suggestion
///                 style: Theme.of(context).textTheme.body1
```